GVFS/GVFS.FunctionalTests/FileSystemRunners/SystemIORunner.cs

```diff
@@ -19,6 +20,17 @@ public override string MoveFile(string sourcePath, string targetPath)
         {
             File.Move(sourcePath, targetPath);
             return string.Empty;
+        }
+
+        public override void CreateFileWithoutClose(string path)
+        {
+            File.Create(path);

```

Comment:
Any ideas on why there was trouble getting this to repro before with `File.Create`?

Reply:
My .NET experience is limited, but I don't see how this inhibits a `close()` on the file descriptor? Surely, `File.Create` returns a `FileStream` which is shortly after GC'd (as it's never stored) and thus closed? Are we relying on GC timing in this test?

Reply:
@wilbaker to answer your question, user error is likely the reason why it didn't repro the first time :-) I tried several times yesterday with repos.

@pmj you are correct that Dispose may eventually get called, but it certainly isn't immediate.  I ran the test several times seeing it fail every time in the old kext.  The alternative of using the low-level windows/mac calls has a lot more overhead and IMHO not worth it since this simpler method causes a repro.

Reply:
OK, sounds good then.

Reply:
> you are correct that Dispose may eventually get called, but it certainly isn't immediate

@jeschu1 do you think we should have the helper method return the file reference so that the test can properly dispose it after running `git status`? (e.g. just have `CreateFile` and `OpenFileAndWrite` and have the test do the close)

That would also prevent the finalizer from kicking in too early in the test.